tekton/release-pipeline-run.yml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: tektoncd-cli-git
+spec:
+  type: git
+  params:
+  - name: revision
+    value: master
+  - name: url
+    value: https://github.com/vdemeester/tektoncd-cli
```

Comment:
should be the official path?

Reply:
ahaha yes, I though I've updated this one before pushing :man_facepalming:

Reply:
Fixed :angel: